Frontend/ui-driver/src/Helpers/Utils.js

```diff
@@ -343,7 +301,7 @@ exports ["setRefreshing"] = function (id){
   }
 }
 
-exports ["setEnabled"] = function (id){
+export const setEnabled = function (id){
   return function (bool){
     if (window.__OS == "ANDROID") {
       cmd = "set_v=ctx->findViewById:i_" + id + ";get_v->setEnabled:b_" + bool + ";"
```

Comment:
`cmd` is not defined anywhere, not sure how it was working before. Will update it in the same PR